src/plugins/chart_expressions/expression_partition_vis/common/utils/consolidate_metric_columns.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { Datatable, DatatableColumn, DatatableRow } from '@kbn/expressions-plugin/common';
+import { getColumnByAccessor } from '@kbn/visualizations-plugin/common/utils';
+import type { ExpressionValueVisDimension } from '@kbn/visualizations-plugin/common';
+
+function nonNullable<T>(value: T): value is NonNullable<T> {
```

Comment:
It would be nice to have a unique reference for this function. It is used all over the places and it is declared in multiple points.

Reply:
(Responded directly)